tests/unit_tests/explainer/test_smart_plotter.py

```diff
@@ -2024,3 +2025,36 @@ def test_correlations_2(self):
         assert len(output.data[0].x) == 3
         assert len(output.data[0].y) == 3
         assert output.data[0].z.shape == (3, 3)
+
+    def test_stability_plot(self):
+        df = pd.DataFrame(np.random.randint(0, 100, size=(15, 4)), columns=list('ABCD'))
+        X = df.iloc[:, :-1]
+        y = df.iloc[:, -1]
+        model = LinearRegression().fit(X, y)
+
+        xpl = SmartExplainer()
+        xpl.compile(x=X,
+                    model=model,
+                    y_pred=y
+                    )
+
+        output = xpl.plot.stability_plot(distribution="none")
+
+        assert len(output.data[0]["x"]) == X.shape[1]
+        assert len(output.data[0]["y"]) == X.shape[1]
+
+    def test_local_neighbors_plot(self):
+        df = pd.DataFrame(np.random.randint(0, 100, size=(15, 4)), columns=list('ABCD'))
+        X = df.iloc[:, :-1]
+        y = df.iloc[:, -1]
```

Comment:
drop y

Reply:
Same as above, don't I need it to fit the model ?

Reply:
Of course sorry